packages/solicitudes/src/components/Sections/InstitucionesAledanas.jsx

```diff
@@ -0,0 +1,87 @@
+import { Grid } from '@mui/material';
+import { DataGrid } from '@mui/x-data-grid';
+import { ButtonAdd } from '@siiges-ui/shared';
+import React from 'react';
+
+const columns = [
+  { field: 'name', headerName: 'Nombre de la institucion', width: 270 },
+  { field: 'time', headerName: 'Tiempo de llegada', width: 100 },
+  { field: 'actions', headerName: 'Acciones' },
+];
+
+const rows = [
```

Comment:
Although you are mocking data I think you should put the data in other file and use functions to generate it